src/include/daos_srv/vos.h

```diff
@@ -81,6 +120,60 @@ vos_dtx_list_cos(daos_handle_t coh, daos_unit_oid_t *oid,
 int
 vos_dtx_list_committable(daos_handle_t coh, struct daos_tx_entry **dtes);
 
+/**
+ * Check whether the specified DTX can be committed or not.
+ *
+ * \param coh	[IN]	Container open handle.
+ * \param dti	[IN]	The DTX identifier.
+ *
+ * \return		See enum dtx_status for kinds of cases.
```

Comment:
I'm not sure I follow what should be done for the various status values

Does pending mean it can be committed?   Committed means nothing needs to be done and init means it's still in progress?

Perhaps this could be written to clarify what is expected behavior after calling this function.   The problem is that the function documentation says it answers the question of whether the DTX can be committed or not.   But what it tells you is the current state of the DTX.   The user is left to try to figure out how to act based on that state but it may not be immediately obvious based on what is in the documentation.

Reply:
OK, I will add more in the comment to make things clearly.